orchagent/orch.cpp

```diff
@@ -1,6 +1,22 @@
+// Copyright 2021 Google LLC
```

Comment:
For existing SONiC files, I think we should remove the Google copyright header. We are not the main author.